src/delete/delete_function.py

```diff
@@ -2,13 +2,24 @@
 import boto3
 import constants
 import custom_exceptions
+import logging
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
 
 def lambda_delete_dbinstance(event, context):
     """Handles deletion of a RDS db instance"""
     region = os.environ['Region']
     rds = boto3.client('rds', region)
     result = {}
     instance_id = event['identifier'] + constants.TEMP_POSTFIX
+
+    logger.info('## ENVIRONMENT VARIABLES')
+    logger.info(region)
+    logger.info('## EVENT')
```

Comment:
can the key and value be combined to a single log?
Or all the variables be formatted and logged in a single log?